stripe/src/main/java/com/stripe/android/model/CardParams.kt

```diff
@@ -91,6 +92,36 @@ internal data class CardParams @JvmOverloads constructor(
         return mapOf(Token.TokenType.CARD to params)
     }
 
+    class Builder : ObjectBuilder<CardParams> {
```

Comment:
interesting - is this manually done? should we consider adding code gen to this sytem?

Reply:
Yeah this was hand-written. I think it would be nice to have codegen but we infrequently add new params models, so I think this works fine for now.